### PR TITLE
feat: Implement CORS for GET and HEAD requests.

### LINKED ISSUE #85: Support CORS
I was tryna make an Angular implementation for a moderation tool, so we can hover over pixels to gather meta data.

Angular was asking for CORs headers and making OPTION requests, this currently isn't supported by the PixelsApi. If it was, development for this implementation and other user's who want to use this style of request would be easier.

### PR DESCRIPTION
**Pull request by Artemis21**
Closes #85.

### CODE REVIEW

File: pixels/pixels.py [hunk 1]
```diff
@@ -42,6 +43,13 @@
     docs_url=None,
     redoc_url=None,
 )
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],
+    allow_methods=["GET", "HEAD"],
+    allow_headers=["*"],
+    allow_credentials=True,
```
Comment (ChrisLovering) [inline pixels/pixels.py:51]: https://fastapi.tiangolo.com/tutorial/cors/ says that "Also, `allow_origins` cannot be set to ['*'] for credentials to be allowed, origins must be specified.".

Does that mean we should remove this line? We don't use cookies in the app for anything other than the auth flow anyway, so no need in enabling this.

Comment (Artemis21) [reply pixels/pixels.py:51]: According to [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Credentials):
> Credentials are cookies, **authorization headers** or TLS client certificates.
Emphasis mine. But yeah it looks like we can't set wildcard origins and allow credentials, so idk.

Comment (Artemis21) [reply pixels/pixels.py:51]: Actually I think that might just refer to not automatically forwarding an `Authorization` header the client used to get the page it is making the request from, but I'm not sure.

Comment (ChrisLovering) [reply pixels/pixels.py:51]: Hmm, should we omit this flag for now then, merge this PR and see what happens when people implement cross-origin services?

Comment (Artemis21) [reply pixels/pixels.py:51]: Sure

**Review by ChrisLovering — APPROVED** [review]

**Comment by Artemis21** [comment]
Tested and this is working without setting `allow_credentials`.

**Comment by ChrisLovering** [comment]
> Tested and this is working without setting `allow_credentials`.

Ah awesome! Nice work :D

**Pull request merged**